### PR TITLE
completed alert component

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**

### CODE REVIEW

File: components/alerts/alert.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    margin-left: 3rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: auto;
+}
+
+.alert-success {
+    background-color: #dff0d8;
```
Comment (rahulk31) [inline components/alerts/alert.css]: Can use variable names for color, it will be helpful in switching colors when you implement dark mode.

**Comment by Sahil25061999** [comment]
Everything looks good the naming as well as the ordering in the css files and as suggested by rahul variables can be used for background colors

**Review by rahulk31 — COMMENTED** [review]
Overall looks good to me. Can be merged. 👍

**Review by kotesh-arya — APPROVED** [review]
Code looks good enough for a merge 👍🏻

File: components/alerts/alert.css (hunk 1 above)
Comment (Adarsh-Patel-Dev) [reply components/alerts/alert.css]: DONE

Comment (Adarsh-Patel-Dev) [reply components/alerts/alert.css]: Done

**Pull request merged**